have large-memory by default:

lilo uses large-memory on initrds which do not fit in 16M,
but it seems large-memory is useful when initrd is >3.5M

diff -p -up lilo-23.2/src/bsect.c.pix lilo-23.2/src/bsect.c
--- lilo-23.2/src/bsect.c.pix	2008-08-22 10:59:49.000000000 +0200
+++ lilo-23.2/src/bsect.c	2008-08-22 16:13:34.000000000 +0200
@@ -854,9 +854,8 @@
 		if (verbose>=2)
 			printf("BIOS data check will include auto-suppress check\n");
 	}
-	/* set to LARGEMEM only if 'large-memory' ist set AND 'small-memory is not set */
-    if (cfg_get_flag(cf_options,"large-memory")
-			&& !cfg_get_flag(cf_options,"small-memory")) {
+	/* set to LARGEMEM if 'small-memory' is not set */
+    if (cfg_get_flag(cf_options,"small-memory")) {
 #ifndef LCF_INITRDLOW
 		bsect.par_1.prompt |= FLAG_LARGEMEM;
 #else
